_includes/site-card.html

```diff
@@ -1,20 +1,40 @@
 {% comment %}
   Single card element for `site-card-list`.
-  
+
   Variables:
     - heading: String heading for a single card.
+    - headingElement: The heading HTML element type.
+    - subheading: String subheading for a single card.
+    - subheadingElement: The subheading HTML element type.
     - body: String body that gets placed in a paragraph tag.
+    - image: The card image.
+    - imageAlt: The alt tag for the image.
     - linkUrl: String for the link url.
     - linkText: String link label.
+    - linkClasses: Classes for the link element.
 {% endcomment %}
-<div class="usa-card__container">
+
+<div class="usa-card__container {{ include.containerClasses }}">
   <header class="usa-card__header">
-    <h2 class="usa-card__heading font-lang-lg">{{ include.heading }}</h2>
+    {% if include.subheading %}
+      <{{ include.subheadingElement | default:"h2" }} class="site-subheading">{{ include.subheading }}</{{ include.subheadingElement | default:"h2" }}>
+    {% endif %}
+    <{{ include.headingElement | default:"h3" }} class="usa-card__heading font-lang-lg">{{ include.heading }}</{{ include.headingElement | default:"h3" }}>
```

Comment:
**question:** Looking at this structure, it looks like the subheading is an `h2` while the heading is an `h3`.

Is this intentional? I'd assume the heading would be a higher level than the subheading

Reply:
It's a good question. In the monthly call card, I specifically wanted the eyebrow subheading to be an h2 because it was a standalone card and I wanted the "Monthly call" text to be a header. For the other cards, I made the subheading element a  `<span>` because the section was organized under a different heading outside of the card grid. Given that, it probably makes more sense to default to an `<span>` for the subheading with the option to use a header if desired. What do you think?
```suggestion
    {% if include.subheading %}
      <{{ include.subheadingElement | default:"span" }} class="site-subheading">{{ include.subheading }}</{{ include.subheadingElement | default:"span" }}>
    {% endif %}
    <{{ include.headingElement | default:"h3" }} class="usa-card__heading font-lang-lg">{{ include.heading }}</{{ include.headingElement | default:"h3" }}>
```

Reply:
Thought about it some more and I went ahead and updated the default element here to "span" in [d698d30](https://github.com/uswds/uswds-site/pull/3115/commits/d698d307da6369c1718ce6530189bc885daf980b). Let me know if you have any objections.

Reply:
Lgtm!